recipes/limereport/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,24 @@
+#include <QCoreApplication>
+#include <QGuiApplication>
```

Comment:
I'd like to know how these are ending up being found. Do they directly come from limereport? I see no requirement to QT in the test package nor are the headers transitive in the main recipe, so wondering about it

Reply:
I missed ```find_package``` for qt from cmakelists but it works locally